pom.xml

```diff
@@ -134,6 +134,31 @@
             </resource>
         </resources>
 
+        <pluginManagement>
```

Comment:
You keep avoiding my questions. Why can't we put checkstyle task as a part of standard build process?